frontend/app/components/contextual-alert.tsx

```diff
@@ -0,0 +1,39 @@
+import type { ReactNode } from 'react';
+
+import { faCheckCircle, faCircleInfo, faExclamationCircle, faExclamationTriangle } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export interface ContextualAlertProps {
+  children: ReactNode;
+  type: 'warning' | 'success' | 'danger' | 'info';
+}
+
+export function ContextualAlert(props: ContextualAlertProps) {
+  const { children, type } = props;
+
+  const alertColor = type === 'warning' ? 'border-l-amber-700' : type === 'danger' ? 'border-l-red-700' : type === 'info' ? 'border-l-cyan-700' : 'border-l-green-700';
+
+  return (
+    <div className={`relative min-w-[290px] pl-[16px] sm:pl-[24px]`}>
+      <div className="absolute left-1.5 top-3 bg-white pt-1 sm:left-3.5">
+        <Icon type={type} />
+      </div>
+      <div className={`overflow-auto border-l-4 ${alertColor} py-2.5 pl-[24px] leading-8`}>{children}</div>
```

Comment:
I'm no Tailwind/CSS expert but should we be specifying specific pixel amounts here (instead of the predefined Tailwind classes that use `rem`)?

CC @sebastien-comeau

Reply:
Honestly, I took that part from SC Labs. I'll change all that I can. Like... [24 px] which already has a pre-defined value in Tailwind